test/test_aio_low_level.py

```diff
@@ -136,3 +137,14 @@ def message_handler(signal):
     assert signal.member == 'SomeSignal'
     assert signal.signature == 's'
     assert signal.body == ['a signal']
+
+
+@pytest.mark.asyncio
+async def test_connection_via_tcp():
+    server = await asyncio.start_server(None, '127.0.0.1', 55556)
+    bus = MessageBus(bus_address="tcp:host=127.0.0.1,port=55556")
```

Comment:
Can you call `connect()` to send a message across?

Reply:
Basically yes. You can specify the callback function and receive the "AUTH EXTERNAL ..." which can be replied with "OK" but then the system got stock (probably due to a lack of the Hello message).

But thats not ultimately necessary to unit test this function because the proof of a network connection seams good enough for me.

Reply:
I'd really like to see a message sent across the connection and successful auth annonymous. #54 for example will add assumptions that the socket is unix that would break this feature and have these tests still pass.

For an example of how to add dbus configuration to the test suites to listen on a tcp address, see the [playerctl dockerfile](https://github.com/altdesktop/playerctl/blob/master/Dockerfile) and test suite which is based on this project.

Reply:
I see the point and I tried to get it working locally - but it did not work before the weekend ended. Somehow the session.conf line order fragility, systemd and even apparmor joined forces to prevent any feeling of success :disappointed: 

Maybe I am also not clever enough or at least not experienced enough in DBus/Docker for this. If you don't want to merge it, it's not a problem either because my company can live with my fork as well. The possibility of a TCP connection is all they needed - I just thought it would be nice to upstream it for other users and to ensure a future for this feature.

Reply:
Ok then this might be a bit more complicated. I can merge it with an issue to create tests if it's a big task. How exactly are you using it? The usage example you gave did some kind of socket forwarding.